mne/io/proj.py

```diff
@@ -222,7 +222,8 @@ def del_proj(self, idx='all'):
         """Remove SSP projection vector.
 
         .. note:: The projection vector can only be removed if it is inactive
-                  (has not been applied to the data).
+                  (has not been applied to the data), unless the channels it
+                  was applied to no longer exist in the data.
```

Comment:
```suggestion
                  was applied to no longer exists in the data.
```

Reply:
It has to be "exist" here:
"unless the **channels** it was applied to no longer **exist** in the data."

Reply:
Agreed what is here is already grammatically correct